### PR TITLE
Set tailcalls to false for net sdk

### PR DESCRIPTION
**Pull request by KevinRansom**
Disable tailcalls by default for Debug configuration in net sdk projects.

Fixes:  4873

### CODE REVIEW

File: src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.props [hunk 1]
```diff
@@ -47,6 +47,7 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
   <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
     <DebugSymbols Condition="'$(DebugSymbols)' == '' ">true</DebugSymbols>
     <Optimize Condition="'$(Optimize)' == '' ">false</Optimize>
+    <Tailcalls Condition="'$(Tailcalls)' == '' ">false</Tailcalls>
```
[the remark this answers is not in the document]
Comment (KevinRansom) [reply src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.props]: @audiochinook, you can set Tailcalls prior to the execution of the props file, you might have a dirs.props that sets tailcalls to true.  The above test is if Condition is set to 'Debug'

[the remark this answers is not in the document]
Comment (auduchinok) [reply src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.props]: @KevinRansom Yes, I get that the property could be set before. My question was about keeping setting the property to true regardless of `Debug` configuration in the previous `PropertyGroup`.
https://github.com/KevinRansom/visualfsharp/blob/b66f851b11bbdda42456d5e4dcc5ea7107dc5840/src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.props#L40

**Comment by dsyme** [comment]
> Enable tailcalls by default for Debug configuration in net sdk projects.

You mean "disable"?

File: src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.props (hunk 1 above)
[the remark this answers is not in the document]
Comment (dsyme) [reply src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.props]: Yes, agreed it's set to `true` above https://github.com/Microsoft/visualfsharp/pull/4876/files#diff-2cc8401da83ac111dbadcb8fb435075eR40

[the remark this answers is not in the document]
Comment (KevinRansom) [reply src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.props]: Oh yeah ...

**Pull request merged**